packages/better-auth/src/init.ts

```diff
@@ -91,7 +92,10 @@ export const init = async (options: BetterAuthOptions) => {
 		return generateId(size);
 	};
 
-	const ctx: AuthContext = {
+	const telemetry = await createTelemetry({ logger, options });
+	telemetry.report();
```

Comment:
No, AI comment is kinda correct. We should handle promises correctly, or there will be an unhandled promise rejection for nodejs

It should be like:

```ts
promise.catch(err => logger.error(err));
```